ui/app/AppLayouts/Chat/ChatColumn/ChatInput.qml

```diff
@@ -276,18 +289,50 @@ Rectangle {
         }
     }
 
+    Row {
+        id: messageLengthLimitContainer
+        anchors.right: parent.right
+        anchors.rightMargin: Style.current.padding
+        anchors.verticalCenter: parent.verticalCenter
+        leftPadding: Style.current.halfPadding
+        anchors.verticalCenterOffset: -18
+
+        StyledText {
+            id: messageLengthLimit
+            text: (messageLimit - txtData.length).toString()
+            color: (messageLimit - txtData.length >= 0) ? Style.current.success : Style.current.danger
+            rightPadding: Style.current.halfPadding
+        }
+    }
```

Comment:
This likely doesn't need a `Row`. A row is used to group multiple items together side-by-side without the need to anchor them to each other. Instead, you can move the anchors to the `StyledText` component instead.

Reply:
It would be nice if we could add a `visible` property to this, that only appears once the message length is within 200 characters of the limit, similar to how Discord does it, ie:
```qml
property int messageLimitVisible: 200

StyledText {
    id: messageLimitLength
    visible: (messageLimit - txtData.length) <= root.messageLimitVisible 
}
```

Reply:
What do you think about also anchoring this to the bottom right of the chat input, similar to how Discord does it? Additionally, we could move the chat buttons to the top. Thoughts @simonricoo?

![Imgur](https://imgur.com/6KgJo1R.png)
![Imgur](https://imgur.com/Sl5KeOg.png)

Reply:
@emizzle In figma its designed with the  new chat input. However you can see that the position is **bottom right**. Check figma for exact dimensions and padding :) 

Also I'm not sure if 

https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=2539%3A430

<img width="1413" alt="Screenshot_2020-09-29_at_09_24_59" src="https://user-images.githubusercontent.com/52249767/94526720-383ddd00-0236-11eb-9936-301c3216a046.png">

Reply:
> It would be nice if we could add a `visible` property to this, that only appears once the message length is within 200 characters of the limit, similar to how Discord does it, ie:
> 
> ```qml
> property int messageLimitVisible: 200
> 
> StyledText {
>     id: messageLimitLength
>     visible: (messageLimit - txtData.length) <= root.messageLimitVisible 
> }
> ```

This looks better than just going below 0. I've made the change

I've also removed the Row and only left the StyledText Component with the Row anchors/padding.

As per the figma I've anchored the buttons and char limit to the bottom right